src/Config/Auth.php

```diff
@@ -374,8 +374,23 @@ public function registerRedirect(): string
 
     protected function getUrl(string $url): string
```

Comment:
The `$url` is no longer a URL. It is a URL or a route name or a route (URI path).
Please add PHPDoc `@param` and the explanation.